unhide-build/index.md

```diff
@@ -0,0 +1,17 @@
+## How do I unhide/hide game builds?
```

Comment:
should we mention edit builds here as well? How do I unhide/hide/edit game builds?